README.md

```
@@ -1,2 +1,67 @@
+[linkedin-badge]: https://img.shields.io/badge/LinkedIn-0077B5?style=for-the-badge&logo=linkedin&logoColor=white
+
+[x]: https://x.com/OrionFinanceAI
+[x-badge]: https://img.shields.io/badge/X-000000?style=for-the-badge&logo=x&logoColor=white
+
+[telegram]: https://t.me/orionfinance_ai
+[telegram-badge]: https://img.shields.io/badge/Telegram-2CA5E0?style=for-the-badge&logo=telegram&logoColor=white
+
+[discord]: https://discord.gg/8bAXxPSPdw
+[discord-badge]: https://img.shields.io/badge/Discord-5865F2?style=for-the-badge&logo=discord&logoColor=white
+
+[docs]: https://docs.orionfinance.ai/
```

markdownlint-cli2 (0.23.2)

[warning] Link and image reference definitions should be needed
Unused link or image reference definition: "docs"

(MD053, link-image-reference-definitions)